lib/google_places/spot.rb

```diff
@@ -411,35 +411,38 @@ def initialize(json_result_object, api_key)
       @vicinity                   = json_result_object['vicinity']
       @lat                        = json_result_object['geometry']['location']['lat']
       @lng                        = json_result_object['geometry']['location']['lng']
-      @viewport                   = json_result_object['geometry']['viewport']
+      @viewport                   = json_result_object['geometry']['viewport'] # 31.05.2016 NOT LONGER IMPLEMENTED
```

Comment:
Thanks for pointing these out - will start work on a `v1.0.0` release that removes all these deprecated fields - will be breaking for the gem, but be good to clean out under a new major version.